tools/wake/describe.cpp

```diff
@@ -87,14 +87,32 @@ static void describe_human(const std::vector<JobReflection> &jobs, bool debug, b
       std::cout << "Stack:";
       indent("  ", job.stack);
     }
-    if (!job.stdout_payload.empty()) {
+
+    std::vector<std::string> stdout_writes;
+    std::vector<std::string> stderr_writes;
+    for (auto &write : job.std_writes) {
+      if (write.second == 1) {
+        stdout_writes.push_back(write.first);
+      }
+      if (write.second == 2) {
+        stderr_writes.push_back(write.first);
+      }
+    }
+
+    if (!stdout_writes.empty()) {
```

Comment:
Did you mean to make these conditional on verbose? I think you always print out stdout and stderr right now

Reply:
ah yeah, nice catch